uGet.yaml

```diff
@@ -0,0 +1,23 @@
+name: uGet
+sources:
+  - type: archive
+    url: https://sourceforge.net/projects/urlget/files/uget%20%28stable%29/2.2.3/uget-2.2.3-1.tar.gz
+    sha256: 11356e4242151b9014fa6209c1f0360b699b72ef8ab47dbeb81cc23be7db9049
+  - type: patch
+    path: gcc10.patch
+  - type: file
+    path: com.ugetdm.uGet.desktop
+  - type: file
+    path: com.ugetdm.uGet.appdata.xml
+
+post-install:
+  - icon_in="pixmaps/icons/hicolor/scalable/apps/uget-icon.svg";
```

Comment:
Just do the svg, no need to install icons of every size unless specifically required.

Reply:
There was a strange Flatpak error which said something along the lines of uknown image type

Reply:
then just do a 512px

Reply:
Done